modules/installation-guide/pages/using-chectl-to-configure-the-checluster-custom-resource-during-installation.adoc

```diff
@@ -0,0 +1,8 @@
+[id="using-{prod-cli}-to-configure-the-checluster-custom-resource-during-installation"]
+// = Using {prod-cli} to configure the `CheCluster` Custom Resource during installation
+:navtitle: Using {prod-cli} to configure the `CheCluster` Custom Resource during installation
```

Comment:
In this case, I wouldn't use 'the' because the '{prod-cli}' attribute refers to 'chectl' (a command-line management tool for Che). Another option is 'the {prod-cli} tool' but that is more wordy.